index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bulls and cows</title>
+    <link rel="stylesheet" href="./styles/main.css">
+  </head>
+  <body>
+    <h1>Bulls and cows</h1>
+    <div class="game">
+      <form class="game__form form">
+        <input
+          type="text"
+          class="form__input"
+          placeholder="Your number"
+          required
+          minlength="4"
+          maxlength="4"
+        >
+        <button class="form__button">Submit</button>
+      </form>
+      <div class="game__output output">
+        <ul class="output__list">
+          <li class="output__item">
+            <span class="output__name">Entered</span>
+            <span class="output__entered"></span>
+          </li>
+          <li class="output__item">
+            <span class="output__name">Result</span>
+            <span class="output__result"></span>
+          </li>
+          <li class="output__item">
+            <span class="output__name">Attempts</span>
+            <span class="output__attempts"></span>
+          </li>
+        </ul>
+        <h2 class="output__gameover"></h2>
+      </div>
+    </div>
+    <script type="text/javascript" src="scripts/main.js"></script>
```

Comment:
better to change the type to "module" or add defer attribute and place script in head tag, after css